docs/patterns/messaging-bridge-content.md

```diff
@@ -0,0 +1,90 @@
+The Messaging Bridge design pattern is a way to integrate disparate systems built on top of different messaging infrastructures.
+
+## Context and problem
+
+Many organizations end up in a situation in which their portfolio of IT systems uses multiple messaging infrastructures, e.g. by way of mergers and acquisitions. Others try to extend their current on-premises systems with cloud-hosted components for ease of maintenance, cost-effectiveness or other reasons. Yet others find themselves in a position where they need to integrate their newer systems with legacy ones. 
```

Comment:
```suggestion
Many organizations end up in a situation in which their portfolio of IT systems uses multiple messaging infrastructures like MSMQ, RabbitMQ, Azure Service Bus, and Amazon SQS. This could be due to mergers and acquisitions, or even by extending their current on-premises systems with cloud-hosted components for ease of maintenance, cost-effectiveness or other reasons.
```

Including examples of technologies for readers who wouldn't immediately know what "messaging infrastructures" refer to. Last sentence also appeared to be a bit overlapping/redundant.